test/Riok.Mapperly.IntegrationTests/Mapper/DerivedMapper.cs

```diff
@@ -20,7 +18,5 @@ public partial class DerivedMapper : BaseMapper
     public partial class DerivedMapper2 : BaseMapper
     {
         public sealed override partial long IntToLong(int value);
-
-        public new partial short IntToShort(int value);
```

Comment:
Instead of just removing the `new` override test, could you add another method testing the `new override?`